mllib/src/main/scala/org/apache/spark/ml/clustering/LDA.scala

```diff
@@ -57,31 +58,32 @@ private[clustering] trait LDAParams extends Params with HasFeaturesCol with HasM
   @Since("1.6.0")
   def getK: Int = $(k)
 
-  /**
-   * Concentration parameter (commonly named "alpha") for the prior placed on documents'
-   * distributions over topics ("theta").
-   *
-   * This is the parameter to a Dirichlet distribution, where larger values mean more smoothing
-   * (more regularization).
-   *
-   * If not set by the user, then docConcentration is set automatically. If set to
-   * singleton vector [alpha], then alpha is replicated to a vector of length k in fitting.
-   * Otherwise, the [[docConcentration]] vector must be length k.
-   * (default = automatic)
-   *
-   * Optimizer-specific parameter settings:
-   *  - EM
-   *     - Currently only supports symmetric distributions, so all values in the vector should be
-   *       the same.
-   *     - Values should be > 1.0
-   *     - default = uniformly (50 / k) + 1, where 50/k is common in LDA libraries and +1 follows
-   *       from Asuncion et al. (2009), who recommend a +1 adjustment for EM.
-   *  - Online
-   *     - Values should be >= 0
-   *     - default = uniformly (1.0 / k), following the implementation from
-   *       [[https://github.com/Blei-Lab/onlineldavb]].
-   * @group param
-   */
+   /*
```

Comment:
ditto: fix

It looks like autoformatting has messed up doc strings elsewhere below as well.  Can you please check through and fix those?